Makefile

```diff
@@ -7,7 +7,11 @@ export GOFLAGS
 GOPROXY ?=
 export GOPROXY
 
+# ENVTEST_K8S_VERSION refers to the version of kubebuilder assets to be downloaded by envtest binary.
+ENVTEST_K8S_VERSION = 1.26
+
 PROJECT_DIR := $(shell dirname $(abspath $(lastword $(MAKEFILE_LIST))))
+ENVTEST = go run ${PROJECT_DIR}/vendor/sigs.k8s.io/controller-runtime/tools/setup-envtest
```

Comment:
Are these new additions used now we've moved things to a separate makefile?

Reply:
Forgot to remove this. Thanks